004-ipfs-in-a-container/src/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM phusion/baseimage:0.9.16
```

Comment:
wondering what the `phusion` image gives us? maybe add a comment for those of us not familiar?